x-pack/plugins/observability/public/data/slo/slo.ts

```diff
@@ -92,7 +92,7 @@ export const sloList: FindSLOResponse = {
     },
     {
       ...baseSlo,
-      id: 'c0f8d669-9177-4706-9098-f397a88173a6',
+      id: 'c0f8d669-9277-4706-9098-f397a88173a6',
```

Comment:
I noticed React was complaining about duplicate ID's when running tests, this fixes that

Reply:
Ah yes good catch I forgot to change the id for this one 🙈